modules/nf-core/gmmdemux/main.nf

```diff
@@ -0,0 +1,65 @@
+
+process GMMDEMUX {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/gmm-demux:0.2.2.3--pyh7cba7a3_0':
+        'biocontainers/gmm-demux:0.2.2.3--pyh7cba7a3_0' }"
+
+    input:
+    tuple val(meta), path(cell_hashing_barcodes,stageAs: "hto_files/*"), path(cell_hashing_matrix,stageAs: "hto_files/*"),path(cell_hashing_features,stageAs: "hto_files/*"),val(hto_names)
+    val csv
+    val simplified_report
+    val examine
+    val ambigous
+    val extract
+    val skip
+
+    output:
```

Comment:
I think all of the report files are missing as (optional) outputs and will not be published properly.